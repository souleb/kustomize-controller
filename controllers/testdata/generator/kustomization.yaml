---
apiVersion: kustomize.toolkit.fluxcd.io/v1beta1
kind: Kustomization
metadata:
  name: app
  namespace: apps
spec:
  interval: 4m0s
  path: ./kustomize
  prune: true
  sourceRef:
    kind: GitRepository
    name: app
  validation: client
  targetNamespace: apps
  patches:
    - patch: |-
        apiVersion: apps/v1
        kind: Deployment
        metadata:
          name: app
        spec:
          template:
            spec:
              containers:
              - name: nginx
                image: nginx:latest
      target:
        name: app
        kind: Deployment