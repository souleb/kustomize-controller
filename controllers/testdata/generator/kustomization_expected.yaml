apiVersion: apps/v1
kind: Deployment
metadata:
  labels:
    environment: ${cluster_env:=dev}
    region: ${cluster_region}
  name: app
  namespace: apps
spec:
  selector:
    matchLabels:
      app: app
  template:
    metadata:
      annotations:
        prometheus.io/port: ${prometheus_port}
        prometheus.io/scrape: ${prometheus_scrape:=false}
      labels:
        app: app
    spec:
      containers:
      - image: nginx:latest
        imagePullPolicy: IfNotPresent
        name: nginx
        resources:
          limits:
            cpu: 2000m
            memory: 512Mi
          requests:
            cpu: 100m
            memory: 64Mi
